runtime/doc/builtin.txt

```diff
@@ -248,6 +339,12 @@ assert_false({actual} [, {msg}])                                *assert_false()*
 		A value is false when it is zero. When {actual} is not a
 		number the assert fails.
 
+                Parameters:
+                  • actual (`any`)
+                  • msg (`any?`)
```

Comment:
In `lua.txt` I was able to align the types as

```suggestion
                Parameters: ~
                  • actual (`any`)
                  • msg    (`any?`)
```

Can we do that here?

Also add a ` ~` at the ends of `Parameters:`

Reply:
> In `lua.txt` I was able to align the types as

I tried this:

```lua
      local s = fmt('- x%-14sx (`%s%s`)', fmt('{%s}', pname), ptype, optional)
      write(util.md_to_vimdoc(s, 16, 18, TEXT_WIDTH))
```

but it looks like `md_to_vimdoc` is scrubbing the extra spaces, so the "alignment" is removed.